api/testing.py

```diff
@@ -25,87 +26,75 @@
     Configuration,
     temp_config,
 )
+
 from api.adobe_vendor_id import AuthdataUtility
 
 class VendorIDTest(DatabaseTest):
-    """Sets up an Adobe Vendor ID integration."""
+    """A DatabaseTest that knows how to set up an Adobe Vendor ID
+    integration.
+    """
 
-    TEST_NODE_VALUE = 114740953091845
     TEST_VENDOR_ID = u"vendor id"
-    TEST_LIBRARY_URI = u"http://me/"
-    TEST_OTHER_LIBRARY_URI = u"http://you/"
-    TEST_OTHER_LIBRARIES  = {TEST_OTHER_LIBRARY_URI: ("you", "secret2")}
```

Comment:
Rather than define constants, which will only work once or twice, we generate the appropriate values for an Adobe Vendor ID or Short Client Token integration based on the library's short_name, which is always different.